### PR TITLE
add shift into delete action

### PR DESCRIPTION
**Pull request by FrancisMengx**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes
Fix for current bug with delete button. 
Repro of the bug:
1. Type in something in input
2. Move the cursor one character back.
3. Hit delete.
Expect:
Only remove the character after the cursor.
Actual:
Remove both the character and highlighted option.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
Should we be committing `yarn.lock` files? cc: @kenotron

**Comment by KevinTCoughlin** [comment]
@FrancisMengx is it possible to add a unit test for this fix?

File: common/changes/office-ui-fabric-react/add-shift-into-delete-action_2018-11-30-21-46.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add shift into delete suggestion.",
+      "type": "minor"
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/add-shift-into-delete-action_2018-11-30-21-46.json]: patch

File: common/changes/office-ui-fabric-react/add-shift-into-delete-action_2018-11-30-21-46.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add shift into delete suggestion.",
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/add-shift-into-delete-action_2018-11-30-21-46.json]: This should be something like `Picker: ....`

**Comment by joschect** [comment]
I'm pretty confused about this. What exactly is this fixing? Also what if I highlight something with shift, but don't have it pressed?

**Comment by FrancisMengx** [comment]
@cliffkoh shift delete is the keyboard short cut in windows deleting something permanently.

**Comment by joschect** [comment]
@cliffkoh I don't know of a designer who owns the people picker/pickers. This change makes sense as right now I would consider it broken, it's really bad that we'll delete suggestions without letting you know if you just press delete.

**Comment by FrancisMengx** [comment]
Do not merge for now I think I found another related issue with the picker.

**Pull request merged**